desk/app/verifier/display.hoon

```diff
@@ -119,16 +248,42 @@
 ++  body
   |^  ;body
         ;div(id "owner"):"{owner}"
-        ;div(id "timestamp"):"{attest-date}"
-        ;div(id "description", class registration-kind)
-          ;a(href id-link)
-            ; {text-description}
+        ;div(id "timestamp"):"Proof of attestation on {nice-date}"
+        ;div(id "full-details")
+          ;h3:"Full Attestation Details"
```

Comment:
May want different copy here for the non-`.full` pages, ie "Partial Attestation Details" or w/e.

Reply:
Sure, I think we can maybe even omit the h3 altogether to avoid over-labeling